src/cljs/pyregence/components/map_controls/weather_station_observation_latest_tool.cljs

```diff
@@ -0,0 +1,231 @@
+
+(defn- $mobile []
+  {:background-color ($/color-picker :bg-color)
+   :box-shadow       (str "1px 0 5px " ($/color-picker :dark-gray 0.3))
+   :color            ($/color-picker :font-color)
+   :height           "290px"
+   :display          "block"
+   :width            "100%"
+   :z-index          "105"})
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Components
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defn- mobile-header []
+  [:div#collapsible-weather-station-header
+   {:style {:align-items      "center"
+            :background-color ($/color-picker :header-color)
+            :display          "flex"
+            :justify-content  "space-between"
+            :padding             "0 1rem"}}
+   [:span {:style {:fill         ($/color-picker :font-color)
+                   :height       "1.5rem"
+                   :margin-right "0.5rem"
+                   :width        "1.5rem"}}
+    [svg/weather-station]]
+   [:label {:style {:font-size "1rem"}}
+    "Wildfire Weather-Station Tool"]
+   [:span {:style {:margin-right "-.5rem"
+                   :visibility   (if (and @!/show-weather-station? @!/mobile?) "visible" "hidden")}}
+    [tool-button :close #(reset! !/show-weather-station? false)]]])
+
+(defn- intro []
+  [:div {:style {:padding "1.2em"}}
+   "Click on a weather station to view it's latest observation. Powered by "
+   [:a {:href   "https://api.weather.gov/"
+        :ref    "noreferrer noopener"
+        :target "_blank"}
+    "Weather.gov"]
+   "."])
+
+(defn- not-found
+  [{:keys [name stationIdentifier]}]
+  [:div
+   [:div {:style {:display         "flex"
+                  :flex-direction  "column"
+                  :justify-content "flex-start"
+                  :width           "100%"
+                  :padding-top     "1rem"
+                  :padding-left    "1rem"}}
+    [:p "No information for this weather station was found. Please check back later or try another station."]
+    [:ul {:style {:padding-inline-start "1rem"}}
+     [:li "Station ID: " stationIdentifier]
+     [:li "Station name: " name]]]])
+
+;;TODO this could share styles with the `not-found` component
+(defn- info [{:keys [stationName stationId timestamp] :as latest-observation} reset-view zoom-weather-station]
+  [:div
+   ;;TODO what do we want to do if the observations go beyond the set window size?
+   [:div {:style {:display         "flex"
+                  :justify-content "flex-start"
+                  :width           "100%"
+                  :padding-top     "1rem"
+                  :padding-left    "1rem"}}
+    ;;TODO when the user hovers on the observation it would be nice to pop up a longer description that includes a longer label
+    [:ul {:style {:padding-inline-start "1rem"}}
+     [:li "Station ID: " stationId]
+     [:li "Station name: " stationName]
+     [:li "Observed at: " (u-time/date-string->iso-string timestamp true)]
```

Comment:
It's just a nice to have, but if this could respect the `show-utc?` atom that would be nice. `show-utc?` keeps track off whether or not a user has UTC or Local time selected (via the time slider).